outputs.tf

```diff
@@ -15,6 +15,7 @@ locals {
   this_tags                         = coalescelist(aws_instance.this.*.tags, [""])
   this_volume_tags                  = coalescelist(aws_instance.this.*.volume_tags, [""])
   this_password_data                = coalescelist(aws_instance.this.*.password_data, [""])
+  this_ipv6_addresses               = compact(coalescelist(aws_instance.this.*.ipv6_addresses, [""]))
```

Comment:
Are you sure that we need `compact` outside?

Reply:
Not 100%, but I looked at the `compact` docs (https://www.terraform.io/docs/configuration/functions/compact.html), and thought that the function to remove unnecessary / empty items in the list couldn't hurt. The same was done for `this_public_ip` and `this_private_ip `, so I was keeping it consistent. If you know it's not needed, I can just remove it.

Reply:
Probably it is not needed, but please verify it and remove it. The rest looks good.